apps/web/app/hooks/features/useKanban.ts

```diff
@@ -73,14 +71,23 @@ export function useKanban() {
 			});
 	};
 
+	const addNewTask = (task: ITeamTask, status: string) => {
+		const updatedBoard = {
+			...kanbanBoard,
+			[status]: [...kanbanBoard[status], task]
+		};
+		console.log('kanban-test-add', updatedBoard);
```

Comment:
@Anishali2 please review EACH line you changed in PR and make sure no console.log and other issues are left. Thank you :)